### PR TITLE
[MX-180] Adds profiles to search entities

### PR DESCRIPTION
**Pull request by ashfurrow**
[I have already submitted a PR to Metaphysics](https://github.com/artsy/metaphysics/pull/2171) to update the persisted queries that 6.2.0 and 6.2.1 are using in production. Since modifying those queries was what broke the behaviour, fixing them now instead of waiting for the next iOS release seemed appropriate.

### CODE REVIEW

File: data/complete.queryMap.json [hunk 1]
```diff
@@ -171,5 +171,11 @@
...
+  "e4098d5f3df8c301e3129df348d14457": "query AutosuggestResultsPaginationQuery(\n  $query: String!\n  $count: Int!\n  $cursor: String\n) {\n  ...AutosuggestResults_results_1jWD3d\n}\n\nfragment AutosuggestResults_results_1jWD3d on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, after: $cursor, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE, PROFILE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n"
```
Comment (ashfurrow) [inline data/complete.queryMap.json]: I'm surprised about all these additions, hmm. I'll take a look and report back.

Comment (ds300) [reply data/complete.queryMap.json]: this file is a minor source of churn and confusion. I'm going to add a ticket to configure relay to make sure it is always canonical.

Comment (ds300) [reply data/complete.queryMap.json]: https://artsyproduct.atlassian.net/browse/MX-183

**Comment by artsyit** [comment]
:rocket: PR was released in v1.21.42 :rocket:

**Pull request merged**